content/docs/user-guide/dvc-files.md

```diff
@@ -143,13 +143,15 @@ stages:
     metrics:
       - performance.json
   training:
-    desc: Training stage description
-    cmd: python train.py
+    desc: Train model with Python
+    cmd:
+      - pip install -r requirements.txt
```

Comment:
is it related? `requirements.txt` should be a dependency. It's not clear why do we make it a single stage. It's probably better to make it a separate stage

Reply:
(I would not include and complicate it here at all)

Reply:
TBH I'm not sure why this isn't already in `master`. It should be (since we introduced multiple commands in dvc.yaml). Maybe a merge error somewhere.

I've extracted this one to #2062 (will roll back here before requesting another review).

Reply:
sounds good (but example is pretty bad to my mind, btw)

Reply:
> example is pretty bad to my mind

Agree, we can discuss in https://github.com/iterative/dvc.org/pull/2062#pullrequestreview-558982138 now 🙂